The build prints the version with the just built nedit.

diff -Nrup a/Makefile b/Makefile
--- a/Makefile	2026-04-28 03:51:53.000000000 +0200
+++ b/Makefile	2026-10-22 17:22:42.524016220 +0200
@@ -32,7 +32,6 @@ all:
 	(cd Xlt;    $(MAKE) -f Makefile.$@ libXlt.a)
 	(cd Microline/XmL;    $(MAKE) -f Makefile.$@ libXmL.a)
 	(cd source; $(MAKE) -f Makefile.$@ nedit nc)
-	@source/nedit -V
 
 # This should not be in the default build, as users may not have Perl
 # installed.  This is only interesting to developers.
